lms/static/coffee/fixtures/send_email.html

```diff
@@ -0,0 +1,9 @@
+<div class="send-email">
+    <div class="request-response msg msg-confirm copy" id="request-response"></div>
```

Comment:
I would include the ul/li nesting structure in the fixture (try to make it more like the actual HTML).